routes/runtime/github.rb

```diff
@@ -64,7 +64,8 @@ class Clover
         scopes = [runner.workflow_job&.dig("head_branch"), repository.default_branch].compact
         entries = repository.cache_entries_dataset
           .exclude(committed_at: nil)
-          .where(key: key, scope: scopes).all
+          .where(key: key, scope: scopes)
+          .order(:version).all
```

Comment:
We can order by `Sequel.desc(:created_at)`

Reply:
Not in the specs, due to the use of transactions.  We could do `Sequel.desc(:created_at), :version`, would you prefer that?

Reply:
Also, FYI, `reverse(:created_at)` is the same as `order(Sequel.desc(:created_at))`

Reply:
Then let's keep it as `version`